companies-manifest.js

```diff
@@ -1205,6 +1206,23 @@ const companies = [
       },
     },
   },
+  {
+    key: 'https://kumparan.com/',
+    similarWebGlobalRank: 2717,
+    name: 'Kumparan',
+    logo: KumparanLogo,
+    style: {},
+    projects: {
+      kumparan: {
+        title: 'Kumparan',
+        link: 'https://kumparan.com/',
+        src: '/screenshots/thumbnails/kumparan.com.png',
```

Comment:
Hey! Thanks for your contribution man!
Can you please compress the picture? 2.7Mb is a little too large for a screenshot for the showcase :/
Also, can you please try and edit out the Ads off the screenshot? It still hasn't been adopted by all the screens but I've seen some contributors doing it in their submission and I really love the idea of letting the design breathe in the showcase!

Looking forward to your updates :)